ChartMogul/Configuration.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Configuration;
+
+namespace ChartMogul.API
+{
+    class Configuration
```

Comment:
This class should probably be deleted. We should not be storing the account token and secret key in Git and we shouldn't be passing them into the calling application from a config file that is local to the client. We should make the consumer of the client pass in the tokens.